tests/test_marker.py

```diff
@@ -103,30 +105,30 @@ def test_dict_as_ujson(self):
 
 
 class EagerMarkerTestCase(MarkerTestCase):
-    def setUp(self):
+    def setUp(self) -> None:
         class MarkerCamera(BaseMarkerCamera):
             marker_dict = MarkerDict.DICT_6X6_50
 
-        self.marker_camera = MarkerCamera(self.MARKER_ID, marker_size=self.MARKER_SIZE)
+        self.marker_camera = MarkerCamera(self.MARKER_ID, marker_size=self.MARKER_SIZE)  # type: ignore
```

Comment:
Why is the type being ignored here? Surely it's obvious what type it should be?

Reply:
`tests/test_marker.py:131: error: Incompatible types in assignment (expression has type "test_marker.MarkerCamera@128", variable has type "test_marker.MarkerCamera@21")`

I have no idea what's going on here :P

Reply:
Worked it out. You had multiple different classes all called MarkerCamera.

9c30a62

Reply:
Worked it out. You had multiple different classes all called MarkerCamera.

9c30a62